vpr/src/route/rr_graph_uxsdcxx_serializer.h

```diff
@@ -1547,12 +1547,24 @@ class RrGraphSerializer final : public uxsd::RrGraphBase<RrGraphContextTypes> {
     }
 
     void finish_load() final {
+        auto& device_ctx = g_vpr_ctx.mutable_device();
```

Comment:
You can remove this line because ``segment_inf_`` already keeps a copy of what you want.

Reply:
Change 
```
for (auto& rr_seg : device_ctx.rr_segments) {
```
to 
```
for (auto& rr_seg : segment_inf_) {
```